pyproject.toml

```diff
@@ -65,7 +65,8 @@ pydata_sphinx_theme = { version = "*", optional = true }
 python = ">=3.7"
 hypercorn = { version = ">=0.12.0", extras = ["trio"] }
 quart = ">=0.18"
-trio = ">=0.9.0"
+trio = ">=0.19.0"
+exceptiongroup = ">=1.0.0"
```

Comment:
This change actually requires [Trio >= 0.22, which is when](https://trio.readthedocs.io/en/stable/history.html#trio-0-22-0-2022-09-28) it converted over to `ExceptionGroup` upstream.